src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2053,6 +2053,30 @@ public void PreviousSubmissionWithError()
             Assert.Throws<InvalidOperationException>(() => CreateSubmission("a + 1", previous: s0));
         }
 
+        [Fact]
+        public void CreateTupleTypeSymbol()
```

Comment:
@AlekseyTs good catch. I mistakenly thought this was in a type that contained only tuple tests.